api/Window.json

```
@@ -7715,45 +7715,141 @@
             "deprecated": false
           }
         },
-        "options_parameter": {
+        "options_behavior_parameter": {
           "__compat": {
-            "description": "<code>options</code> parameter",
+            "description": "<code>options.behavior</code> parameter",
             "support": {
               "chrome": {
-                "version_added": "45"
+                "version_added": "35"
```

This is based on https://source.chromium.org/chromium/chromium/src/+/d5703dc81f333467a9fd2b929ec56d1f8c2410ca but "These behavior changes are protected by a runtime flag" in the commit message.

Based on http://software.hixie.ch/utilities/js/live-dom-viewer/?saved=9636 and some manual tweaking of the test to see the error, I think it was all enabled in Chrome 61:

```suggestion
                "version_added": "61"
```

Can you mirror the same change throughout?